numpy/lib/function_base.py

```diff
@@ -907,8 +907,8 @@ def gradient(f, *varargs, **kwargs):
     Returns
     -------
     gradient : ndarray
-        N arrays of the same shape as `f` giving the derivative of `f` with
-        respect to each dimension.
+        `list` of `ndarray` of the same shape as `f` giving the derivative 
```

Comment:
It may also be a good idea to keep the reference to `N`, which is used everywhere else in the docstring, e.g. "List of N ndarrays..."